app/models/page.rb

```diff
@@ -55,9 +55,9 @@ def navigation?
     page_style == 'side-nav'
   end
 
-  # @return [ContentfulModel::Asset]
+  # @return [ContentfulModel::Asset, nil]
   def thumbnail
-    return ::OpenStruct.new(url: 'https://placehold.co/380x254/347ca9/FFFFFF/png') if fields[:image].blank?
+    return nil if fields[:image].blank?
```

Comment:
You could remove `nil` if you like and it'd behave the same because it is implicit